Monika-After-Story/MonikaModDev
# Typo fixes

Let's make Monika proud of us!

### Testing
- verify all reported types are fixed
- verify correctness
- verify general stability
- test update script
